acquia/cli
# DX-2666: Test all exception listeners

Motivation
----------
Only one exception listener is being tested right now, and it's not scalable to test other exception listeners like that because the existing test bootstraps the whole application.

Proposed changes
---------
Unit test _all_ exception listeners using a unit test (no bootstrap). Also move misc test cases into a separate folder from the abstract bases, just for organization's sake.

Alternatives considered
---------
Build the whole application to test listeners end-to-end, but this is going to be really ugly and slow, and doesn't add much in the way of coverage.

Testing steps
---------
How can we replicate the issue and verify that this PR fixes it?

1. Just verify that code coverage goes up.
